src/Components/Authentication/__tests__/Desktop/ForgotPasswordForm.test.tsx

```diff
@@ -2,55 +2,77 @@ import { ForgotPasswordForm } from "Components/Authentication/Desktop/ForgotPass
 import { mount, shallow } from "enzyme"
 import React from "react"
 
-describe("ResetPasswordForm", () => {
-  xit("calls handleSubmit with the right params", () => {
-    const props = {
+jest.mock("sharify", () => ({ data: { RECAPTCHA_KEY: "recaptcha-api-key" } }))
+
+describe("ForgotPasswordForm", () => {
+  let props
+
+  beforeEach(() => {
+    props = {
       handleSubmit: jest.fn(),
+      values: { email: "foo@bar.com" },
     }
+    window.grecaptcha.ready.mockClear()
+    window.grecaptcha.execute.mockClear()
+  })
 
-    const values = {
-      email: "foo@bar.com",
-    }
+  const getWrapper = (passedProps = props) => {
+    return mount(<ForgotPasswordForm {...passedProps} />)
+  }
 
-    const wrapper = shallow(
-      <ForgotPasswordForm values={values} handleSubmit={props.handleSubmit} />
-    )
+  describe("onSubmit", () => {
+    it("calls handleSubmit with expected params", done => {
+      const wrapper = shallow(<ForgotPasswordForm {...props} />)
+      const formik = wrapper.dive().instance() as any
+      formik.submitForm()
+      wrapper.update()
 
-    const formik = wrapper.dive().instance() as any
-    formik.submitForm()
+      setTimeout(() => {
+        expect(props.handleSubmit).toBeCalledWith(
+          {
+            email: "foo@bar.com",
+          },
+          formik.getFormikActions()
+        )
+        done()
+      })
```

Comment:
Something like
```js
import { showRecaptcha } from 'Utils/showRecaptcha'
...
jest.mock('Utils/showRecaptcha')
...
expect(showRecaptcha).toHaveBeenCalledWith(...)
```

Reply:
The async behavior relates to the Formik submit rather than the recaptcha functionality.

Reply:
No prob 👍